### PR TITLE
Feature/te 260 test stop stepwise in vm

### PR DESCRIPTION
**Pull request by kpanier**
Changing property rules for headless runner. 
Command line arguments now overrides workspace settings in the headless runner mode.
The debug port is mapped in the vagrant file of the tests in the test-editor-test repository.

### CODE REVIEW

File: core/org.testeditor.core/src/main/java/org/testeditor/core/headless/HeadlessTestRunnerApplication.java [hunk 1]
```diff
@@ -240,18 +239,8 @@ public TestStructure getTestStructureToExecute(String[] args)
 	 */
 	public void initializeSystemConfiguration() throws BackingStoreException, IOException {
 		TestEditorConfigurationService testEditorConfigService = getService(TestEditorConfigurationService.class);
-		String waittime = System.getProperty(TestEditorGlobalConstans.DEFINE_WAITS_AFTER_TEST_STEP);
-		String browserPath = System.getProperty(TestEditorGlobalConstans.PATH_BROWSER);
-		testEditorConfigService.exportGlobalVariablesToSystemProperties();
+		testEditorConfigService.exportGlobalVariablesToSystemProperties(false);
```
Comment (franzbecker) [inline core/org.testeditor.core/src/main/java/org/testeditor/core/headless/HeadlessTestRunnerApplication.java]: I think the readability would improve by refactoring the variable name to `configService`, same for `TestEditorConfigurationServiceImplIntTest`

File: core/org.testeditor.core/src/main/java/org/testeditor/core/services/impl/TestEditorConfigurationServiceImpl.java [hunk 2]
```diff
@@ -64,12 +64,18 @@ protected String fixWhiteSpaceOnPreferenceString(String replaceStr) {
 	}
 
 	@Override
-	public void exportGlobalVariablesToSystemProperties() throws BackingStoreException {
+	public void exportGlobalVariablesToSystemProperties(boolean overRide) throws BackingStoreException {
```
Comment (franzbecker) [inline core/org.testeditor.core/src/main/java/org/testeditor/core/services/impl/TestEditorConfigurationServiceImpl.java]: I don't see a reason for camel-case here, `override` would be just fine.

**Comment by franzbecker** [comment]
It would be nice if we had a unit test for `TestEditorConfigService.exportGlobalVariablesToSystemProperties` for the two cases of `override` as well.

**Comment by kpanier** [comment]
Fixing:
- the method parameter
- rename service variable
- adding integration tests

**Pull request merged**